### DISCUSSION TITLE
Linear advance board options?

### DISCUSSION THREAD

**Discussion by cristian-rivera** [Jyers/Marlin#890] [Q&A] [answered]
I have an Ender 3 V2 with a 4.2.2 board. Is there a board on the market that is a drop in replacement without having to modify display wires that also supports linear advance?

I know SKR exists, but not sure if I want to go down the rewiring route yet!

**Comment by smooreace** [comment]
Im really leaning toward moving to an SKR board myself! Possibly when this
newest board becomes more readily available. Linear advance seems like the
best option for making a v2 the best it can be

**Comment by Jyers** [comment, marked as answer]
I would highly encourage you to take a look at #707. Linear advance is totally doable with a 4.2.2 board is your willing to do a little modding!
